docs/linux/ibus-keyman.md

```diff
@@ -6,28 +6,27 @@ Source code for Keyman engine for IBus is in [linux/ibus-keyman](../../linux/ibu
 
 ## Requirements
 
-You need autoconf, autopoint, gettext, automake and libtool to generate the build system.
-
-Run `./autogen.sh` to run them.
+meson (>= 0.53)
 
 ## Building
 
 ```bash
-./autogen.sh
-./configure
-make
-sudo make install
+./build.sh all
+./build.sh test
+sudo ./build.sh install
 ```
 
 For a debug build:
 
 ```bash
-./configure CPPFLAGS=-DG_MESSAGES_DEBUG CFLAGS="-g -O0" CXXFLAGS="-g -O0"
+CPPFLAGS=-DG_MESSAGES_DEBUG CFLAGS="-g -O0" CXXFLAGS="-g -O0" ./build.sh --debug clean config build
```

Comment:
Wondering why those flags aren't added to build.sh?

Reply:
Good point. Done. (Turns out we don't have to specify `-g` since that's the default with buildtype `debug`)